cajita/src/Cajita_FastFourierTransform.hpp

```diff
@@ -553,17 +422,25 @@ class HeffteFastFourierTransform
                 auto iw = i - own_space.min( Dim::I );
                 auto jw = j - own_space.min( Dim::J );
                 auto kw = k - own_space.min( Dim::K );
-                work_view( iw, jw, kw ) = copyFromKokkosComplex(
-                    x_view( i, j, k, 0 ), work_view( iw, jw, kw ) );
+                work_view( iw, jw, kw, 0 ) = x_view( i, j, k, 0 );
+                work_view( iw, jw, kw, 1 ) = x_view( i, j, k, 1 );
             } );
 
         if ( flag == 1 )
         {
-            _fft->forward( _fft_work.data(), _fft_work.data(), scale );
+            _fft->forward(
+                reinterpret_cast<std::complex<Scalar>*>( _fft_work.data() ),
+                reinterpret_cast<std::complex<Scalar>*>( _fft_work.data() ),
+                scale );
+            // _fft->forward( _fft_work.data(), _fft_work.data(), scale );
         }
         else if ( flag == -1 )
         {
-            _fft->backward( _fft_work.data(), _fft_work.data(), scale );
+            _fft->backward(
+                reinterpret_cast<std::complex<Scalar>*>( _fft_work.data() ),
+                reinterpret_cast<std::complex<Scalar>*>( _fft_work.data() ),
+                scale );
+            // _fft->backward( _fft_work.data(), _fft_work.data(), scale );
```

Comment:
Should this commented out code be removed?

Reply:
Removed. Thanks for catching that